README.md

```diff
@@ -101,6 +101,22 @@ that iterates through them:
 order.calculate_total_price(double(:price => 1.99), double(:price => 2.99))
 ```
 
+### Stubbing a chain of methods
+
+You can use `receive_message_chain` in place of `receive` in certain circumstances to stub a chain of messages:
```

Comment:
> in certain circumstances

This makes it sound like these code snippets only work _sometimes_; you have to know if you're in one of the "certain circumstances" where these work.

These code snippets always work, though; (assuming you haven't disabled the newer stubbing syntax), so I'd remove the phrase "in certain circumstances" entirely.